components/ShareModal/ShareModal.tsx

```diff
@@ -291,14 +242,14 @@ const ShareModal = ({
                 <Collapse.Panel header="How does it work?" key="1">
                   <div className="text-justify">
                     <a className="font-medium">
-                      Copy your share code with <i className="bi bi-copy" />
+                      Copy your share code with <i className="bi bi-copy"/>
                     </a>{" "}
```

Comment:
`<a>` -> `<span>`
I know this was already there, but it was a mistake of mine 😅 This makes the browser think the text is a link and highlight it when hovering.